prisma/schema.prisma

```diff
@@ -12,9 +12,93 @@ datasource db {
 }
 
 model Pokemon {
-    id          Int       @id @default(autoincrement())
-    name        String
-    art         String
-    type1       String
-    type2       String?
-}
\ No newline at end of file
+  id             Int              @id @default(autoincrement())
+  name           String
+  art            String
+  type1          String
+  type2          String?
+  ratingAvg      Int              @default(1)
+  ratingCount    Int              @default(0)
+  createdAt      DateTime         @default(now())
+  updatedAt      DateTime         @updatedAt
+  userTeamMember UserTeamMember[]
+  rating         Rating[]
+}
+
+model User {
+  id        Int      @id @default(autoincrement())
+  username  String
+  password  String
+  email     String
+  admin     Boolean  @default(false)
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+  profile   Profile?
+
+}
+
+model Profile {
+  id             Int              @id @default(autoincrement())
+  user           User             @relation(fields: [userId], references: [id])
+  userId         Int              @unique
+  createdAt      DateTime         @default(now())
+  updatedAt      DateTime         @updatedAt
+  userTeamMember UserTeamMember[]
+  rating         Rating[]
+  post           Post[]
+  Comment        Comment[]
```

Comment:
comment is capitalised

Reply:
Ah that was something the schema did on its own, I forgot to correct it. Will do now